framework/test/mocha/functional/http/get/node/transactions_ready.js

```diff
@@ -318,6 +318,24 @@ describe('GET /api/node', () => {
 					});
 				});
 
+				describe('nonce', () => {
+					it('sorted by nonce:asc should be ok', async () => {
+						return ReadyEndpoint.makeRequest({ sort: 'nonce:asc' }, 200).then(
```

Comment:
return ReadyEndpoint
						.makeRequest({ sort: 'nonce:asc' }, 200)
						.then(res => {
							const values = _.map(res.body.data, 'nonce').map(value => {
								return parseInt(value, 10);
							});

							expect(_(_.clone(values)).sortNumbers('asc')).to.be.eql(values);
						});
And same for below test?

Reply:
It seems like above tests similar to this is written in the same way

Reply:
ya it was not adding much value..